bin/dependencies/build_opentelemetry-swift.sh

```diff
@@ -1,6 +1,16 @@
 REPO_DIR=$1
 BUILD_DIR=$2
 
+# We replace the `package` keyword because it's not supported by the `create-xcframework` command.
+# We tried using "OTHER_SWIFT_FLAGS" and setting the package and module names, but it ends up creating problems
+# since we are building two frameworks at the same time.
+# Until the `package` keyword is supported directly, we'll replace the `package` keyword with just `public`
+# so the frameworks compile.
+# We realize this exposes something that is not meant to be exposed, we think it's harmless.
+
+grep -rl 'package static' "$REPO_DIR/Sources/" | xargs sed -i '' 's/package static/public static/g'
```

Comment:
We should add a SwiftLint rule that prevents `package` access until we can support it directly. This leads to an inconsistent interface depending on if we distribute the xcframework or we distribute using source. That will lead to confusion in the future.

For now, we should change/commit the code changes that is using the `package` keyword so our interfaces are consistent, even if they're more public than we would hope

Reply:
Just realized this is specifically for the `opentelemetry-swift` dependency. How has this worked for previous releases?

Reply:
This is affecting the OpenTelemetryApi and OpenTelemetrySdk libraries, not Embrace.
The only alternative to this that we could think of was to fork them.